application/server/controllers/spells.go

```diff
@@ -175,3 +175,27 @@ func (repository *Repos) UpdateSpell(c *gin.Context) {
 
 	c.JSON(http.StatusAccepted, &spell)
 }
+
+// Note that sending empty body gets all spells
+func (repository *Repos) GetFilteredSpells(c *gin.Context) {
+	var spellFilters models.FilterSpells
+	err := c.ShouldBindJSON(&spellFilters)
+	if err != nil {
+		c.AbortWithStatusJSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+
+	var spells []models.Spell
+	err = repository.SpellDb.Where(&models.Spell{LevelReq: spellFilters.LevelReq, ClassReq: spellFilters.ClassReq}).Find(&spells).Error
```

Comment:
Hmm should it be that you can get all the spells that are at your level or lower?
I think this only gets spells at your level

Reply:
This is fixed now